src/test/javascript/integration/channel_creation_zeroBytes_spec.js

```diff
@@ -13,52 +17,52 @@ var testName = __filename;
  *
  */
 describe(testName, function () {
-    var returnedBody;
-
+    var returnedBody; // TODO: is this actually being used?
```

Comment:
It looks like an over zealous memory optimization from the past :) Since `parse` is being used directly anyway I would just get rid of it.